src/api/transactionSubmission/simulate.ts

```diff
@@ -41,19 +41,19 @@ export class Simulate {
   /**
    * Simulate a multi agent transaction
    *
-   * @param args.signerPublicKey The signer public key
+   * @param args.signerPublicKey optional. The signer public key
    * @param args.transaction An instance of a raw transaction
-   * @param args.secondarySignersPublicKeys An array of the secondary signers public keys
+   * @param args.secondarySignersPublicKeys An array of the secondary signers public keys, each of which can be optional
```

Comment:
lets add some more info, something like
```suggestion
   * @param args.secondarySignersPublicKeys An array of the secondary signers public keys, each of which can be optional to support the public key skip checking
```

Reply:
Added some info